DQM/EcalMonitorTasks/python/EcalMonitorTask_cff.py

```diff
@@ -0,0 +1,11 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQM.EcalMonitorTasks.EcalMonitorTask_cfi import *
+
+# Customization to run the CPU vs GPU comparison task if the job runs on a GPU enabled machine
+from Configuration.ProcessModifiers.gpu_cff import gpu
+from DQM.EcalMonitorTasks.GpuTask_cfi import ecalGpuTask
+
+gpu.toModify(ecalGpuTask.params, runGpuTask = cms.untracked.bool(True))
```

Comment:
It seems that by setting `runGpuTask` to `True` here by default this will run the GPU to CPU comparison by default in the offline DQM if the `gpu` modifier is given. I do not think this is what we want since it implies to run the CPU and the GPU reconstruction.

As discussed in https://github.com/cms-sw/cmssw/issues/35879 we should use a different modifier that is only given when the GPU vs. CPU comparison should be done. @jfernan2 does such a modifier exist already from DQM? In the issue mentioned it was proposed to call it `gpu-validation`.